Code/Tools/ProjectManager/CMakeLists.txt

```diff
@@ -89,6 +89,7 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED)
                 3rdParty::Qt::Widgets
                 3rdParty::Python
                 3rdParty::pybind11
+                AZ::AzCore
```

Comment:
Maybe I misunderstand how CMake works, but AZ::AZTest declares a public build dependency on AZ::AzCore so I don't see why ProjectManager tests also need to do so.  What problem did this solve?

Reply:
Removed, it doesn't do anything was just not completely removed when testing ways to make some of the tests work, my mistake.